### DISCUSSION TITLE
Meeting Agenda 4/11

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#374] [Meeting Agendas/Minutes]
1. Review tasks
2. Meet with Darian

**Comment by moreno-michael** [comment]
Meeting minutes 4/11

1. Task Review
- Issue #1608 completed
- Remaining tasks still in progress
- Issues occuring with Michael's dev environment (Hardware limitation)
2. Meet with Darian
- Task Review
- No real further discussion
